android/src/test/java/com/shopify/reactnative/flash_list/AutoLayoutShadowTest.kt

```diff
@@ -91,7 +91,7 @@ internal class AutoLayoutShadowTest {
         }
 
         //Item right of visible window
-        arrayOf<CellContainer>(getRect(0, 0, 100, 100, 0), getRect(550, 120, 100, 100, 1)).let {
+        arrayOf<CellContainer>(getRect(0, 0, 100, 100, 0), getRect(850, 120, 950, 220, 1)).let {
```

Comment:
Yes because the neighbours should now be adjusted while this test checks the opposite. In this test the neighbour was visible but the previous cell wasn't. With new code, the neighbour will be adjusted even in this case.